simulation/model.py

```diff
@@ -26,6 +27,7 @@ def __init__(
         annual_renovation_rate,
         household_num_lookahead_years,
         heating_system_hassle_factor,
+        intervention,
```

Comment:
It probably makes a bit more sense to pass in a `Optional[InterventionType]` (later perhaps being an `Optional[List[InterventionType])`) here rather than a string (which gets transformed under the hood).